int/config/check.go

```diff
@@ -0,0 +1,126 @@
+package config
+
+import (
+	"crypto/x509"
+	"fmt"
+	"path/filepath"
+
+	"github.com/massalabs/station/int/configuration"
+	"github.com/massalabs/station/pkg/certificate"
+	"github.com/massalabs/station/pkg/certificate/store"
+	"github.com/massalabs/station/pkg/logger"
+	"github.com/massalabs/station/pkg/nss"
+)
+
+// failureConsequence is the consequence of a failure to add the CA to NSS.
+const failureConsequence = "Station will only work using http, or you will have to add the CA to your browser manually."
+
+// Check performs a check on the configuration.
+func Check() error {
+	caRootPath, err := configuration.CAPath()
+	if err != nil {
+		return caNonBlockingError("failed to get CA path", err)
+	}
+
+	certPath := filepath.Join(caRootPath, configuration.CertificateAuthorityFileName)
+
+	err = checkCertificate(certPath)
+	if err != nil {
+		return nonBlockingError("Error while checking certificate", failureConsequence, err)
+	}
+
+	err = checkNSS(certPath)
+	if err != nil {
+		return nonBlockingError("Error while checking NSS", failureConsequence, err)
+	}
+
+	return nil
+}
+
+// nonBlockingError logs a non blocking error.
```

Comment:
I would add something about the fact that we always return a `nil` error such as:
```suggestion
// nonBlockingError logs a non blocking error. It always returns a `nil` error to be used in `return`.
```